avm/ptn/azd/aks/main.bicep

```diff
@@ -296,7 +294,7 @@ module containerRegistry 'br/public:avm/res/container-registry/registry:0.4.0' =
   }
 }
 
-module keyVault 'br/public:avm/res/key-vault/vault:0.7.1' = {
+module keyVault 'br/public:avm/res/key-vault/vault:0.7.1' = if (enableTelemetry) {
```

Comment:
Not the correct fix. Please don't add the telemetry parameter as a condition, but just pass the parameter in the params block through to the linked module.

Reply:
I have updated by your comments.